### DISCUSSION TITLE
chrome://theme/IDR_PRODUCT_YIFF

### DISCUSSION THREAD

**Discussion by Aedon** [Alex313031/Thorium-Win-AVX2#83] [General]
I wonder what opening that url would do (listed as chrome://yiff in the chrome://about page)
😁

**Comment by Alex313031** [comment]
@Aedon Lol we got another one that found the easter egg.
That makes two.
Might remove it soon though because people might not like it.
Might just remove the link on chrome://about, but leave the image embedded in the resources.pak file lolol
